tests/validation/tests/v3_api/test_workload.py

```diff
@@ -858,3 +858,306 @@ def setup_project_by_role(role, remove_resource):
 
     else:
         return None, None, None, None
+
+# --------------------- rbac tests by workload type -----------------------
+@if_test_rbac
+@pytest.mark.parametrize("role", [CLUSTER_MEMBER,
```

Comment:
Can we get this role list as a global variable and refactor the parameterized test methods parameter with that variable?